sdk/eventhub/event-processor-host/package.json

```diff
@@ -22,7 +22,7 @@
   "module": "dist-esm/src/index.js",
   "types": "./typings/event-processor-host.d.ts",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=12.0.0"
```

Comment:
No version bump for this legacy package?

Reply:
Nah, no plans on releasing anything here.